### PR TITLE
Add notification email when password is changed

### PR DESCRIPTION
**Pull request by rowanseymour**
And simplify notification emails by converting the target URL to absolute in the view instead of the template

### CODE REVIEW

File: temba/notifications/models.py [hunk 1]
```diff
@@ -184,21 +184,22 @@ class Notification(models.Model):
     incident = models.ForeignKey(Incident, null=True, on_delete=models.PROTECT, related_name="notifications")
 
     @classmethod
-    def create_all(cls, org, notification_type: str, *, scope: str, users, **kwargs):
+    def create_all(cls, org, notification_type: str, *, scope: str, users, medium: str = MEDIUM_UI, **kwargs):
```
Comment (rowanseymour) [inline temba/notifications/models.py:187]: some tweaking here as this will be the first email only notification type

**Review by norkans7 — APPROVED** [review]

**Comment by rowanseymour** [comment]
Have also added a notifcation type for when a user's email is changed. I thought about whether that should be combined with the password changed notification but kept them separate because:
  1. Changing your password and email at the same time isn't really that common
  2. There should be a different pipeline for changing email that requires verification

**Comment by rowanseymour** [comment]
@norkans7 ah good point... it should be the old one.... hmm

**Comment by rowanseymour** [comment]
@norkans7 updated so that email changed notification should go to the old email address - had to add `email_address` to `Invitation`

**Comment by norkans7** [comment]
I think you mean `Notification` instead of `Invitation`.
Looks good

**Pull request merged**